src/lephare/process.py

```diff
@@ -219,3 +222,78 @@ def table_to_data(config, input, col_names=None, standard_names=False):
     flux[mask] = -99.0
     flux_err[mask] = -99.0
     return id, flux, flux_err, context, zspec, string_data
+
+
+def load_sed_list(path_to_list, object_type, absolute_paths=False):
+    """Take a sed list and move it with the sed files to the work directory.
+
+    We want to be able to load SED lists from other sources. These may use
+    absolute paths or relative to the list file. They must adopt the lepahre
+    standard which is relative to the sed/$TYPE/ directory
+
+    Parameters
+    ==========
+    path_to_list : str
+        The path to the SED list file.
+    object_type : str
```

Comment:
The numpy docstring style can accommodate this like so: 
```suggestion
    object_type : {'STAR', 'GAL', 'QSO'}
```